hw1/02-palindrome.js

```diff
@@ -1,3 +1,23 @@
 const elem = document.querySelector('input');
-
+const resultText = document.getElementById('resultText');
 elem.addEventListener('input', handleInput);
```

Comment:
You could have defined the "handleInput" function before it is called to avoid any reference errors

Reply:
Thanks! Done.